### PR TITLE
feat: Add space delegation portal schema

### PR DESCRIPTION
**Pull request by samuveth**
The `delegation` wasn't used are far as I can tell, so I removed it as well.

### CODE REVIEW

File: src/schemas/space.json [hunk 1]
```diff
@@ -234,8 +234,31 @@
...
+            ]
+          },
+          "contract": {
+            "type": "string",
+            "format": "address",
+            "title": "Contract address",
+            "description": "The address of your delegation contract",
+            "examples": ["0x3901D0fDe202aF1427216b79f5243f8A022d68cf"]
+          },
+          "subgraphUrl": {
```
Comment (bonustrack) [inline src/schemas/space.json:253]: Depending on the standard the API may not be a subgraph, I would call it `delegationApi` instead.

File: src/schemas/space.json [hunk 2]
```diff
@@ -234,8 +234,31 @@
           "required": ["name"],
           "additionalProperties": false
         },
-        "delegation": {
```
Comment (bonustrack) [inline src/schemas/space.json:237]: We shouldn't remove this, may break some stuff

File: src/schemas/space.json [hunk 3]
```diff
@@ -234,8 +234,31 @@
...
-          "type": "boolean"
+        "delegationPortal": {
+          "standard": {
```
Comment (bonustrack) [inline src/schemas/space.json:238]: I would call this "delegationType" similar than on SX-UI. The type may not be a real "standard".

File: src/schemas/space.json (hunk 2 above)
Comment (samuveth) [reply src/schemas/space.json:237]: It's not used anywhere and it's not a required field. How can I break anything?

File: src/schemas/space.json [hunk 4]
```diff
@@ -234,8 +234,31 @@
...
+            "format": "address",
+            "title": "Contract address",
+            "description": "The address of your delegation contract",
+            "examples": ["0x3901D0fDe202aF1427216b79f5243f8A022d68cf"]
+          },
+          "delegationApi": {
+            "type": "string",
+            "format": "uri",
+            "title": "Subgraph URL",
+            "description": "The URL of your delegation subgraph",
```
Comment (samuveth) [inline src/schemas/space.json:257]:
```suggestion
            "title": "Delegation API",
            "description": "The URL of your delegation API (e.g a subgraph)",
```

File: src/schemas/space.json [hunk 5]
```diff
@@ -234,8 +234,31 @@
           "required": ["name"],
           "additionalProperties": false
         },
-        "delegation": {
-          "type": "boolean"
+        "delegationPortal": {
+          "delegationType": {
+            "type": "string",
+            "title": "Standard",
+            "description": "The standard used by your delegation contract",
```
Comment (samuveth) [inline src/schemas/space.json:241]:
```suggestion
            "title": "Delegation type",
            "description": "Specify the type of delegation that you are using",
```

File: src/schemas/space.json (hunk 2 above)
Comment (ChaituVR) [reply src/schemas/space.json:237]: Yea `delegation` was added before when we thought of refactoring delegation but we didn't work on it anymore, I think it should be fine, couldn't find any references with this, not even on hub

**Review by bonustrack — APPROVED** [review]
utACK

**Comment by tamebadger** [comment]
Screenshot 2023-06-07 at 22 22 36
seeing this error popping up everywhere, is it perhaps related ?

**Comment by ChaituVR** [comment]
Reverting this for now

**Pull request merged**